apps/cowswap-frontend/src/modules/swap/containers/TradeBasicDetails/index.tsx

```diff
@@ -29,11 +29,10 @@ export function TradeBasicDetails(props: TradeBasicDetailsProp) {
 
   // trades are null when there is a fee quote error e.g
   // so we can take both
-  const feeFiatValue = useHigherUSDValue(trade?.fee.feeAsCurrency || fee).value
+  const feeFiatValue = useFiatAmount(fee).value
   const isEoaEthFlow = useIsEoaEthFlow()
   const isWrapOrUnwrap = useIsWrapOrUnwrap()
 
-  const showRowFee = trade || fee
```

Comment:
Because we don't need `trade` to display a fee value anymore and `fee` is always present:
```
const showTradeBasicDetails = (isFeeGreater || trade) && fee

  {showTradeBasicDetails && (
    <TradeBasicDetails
      allowedSlippage={userAllowedSlippage}
      isExpertMode={isExpertMode}
      allowsOffchainSigning={allowsOffchainSigning}
      trade={trade}
      fee={fee}
    />
```